### PR TITLE
NXP MR-VMU-RT1176 flightcontroller update

### PR DESCRIPTION
**Pull request by hamishwillee**
This replaces https://github.com/PX4/PX4-user_guide/pull/3397 which would not accept subedits. FYI @dk7xe @mrpollo and @igalloway 

@dk7xe I had to create this because your branch wasn't editable (by me).
The main changes are editorial - I found the introduction a bit confusing so I have moved some bits around. I think it now makes things a bit more clear that this is reference design board consisting of a carrier and an FMU.

I also 
- made the images bigger.
- Made the discontinued markup work properly for the older board. It is still manufacturer supported, but it moves out of that page because it is better for users to not consider discontinued boards when selecting. If they open the board page they will see it is discontinued.
- Retitled to "NXP MR-VMU-RT1176 Flight Controller". Adding _Flight Controller_ makes it obvious what we're talking about in searches etc.
- Either hid TBDs or replaced with "Use this". There is no need to expose that it is TBC, particularly for the wiring diagram when the wiring is largely the same as other pixhawk-connector standard FCs.

@dk7xe @igalloway
- This is available now?
- Is it true that you can install firmware with current release QGC and PX4 v1.15? Does it work with earlier versions? We should say.
- It would be good to have a link to the **FMUv6X-RT open standard** since this is mentioned several times. Do we have one?

### CODE REVIEW

**Comment by hamishwillee** [comment]
FYI what I normally do is create a branch for my edits rather than push up my main branch. So assuming your repo is `origin` and the PX4 repo is `upstream`, I'd start my work like this:

```
# get main to match upstream
git checkout main
git fetch upstream main
git pull upstream main

# Branch to a sensible name other than main
git checkout -b pr_something
# Do work, add commits
#Perhaps rebase against upstream again
git fetch upstream main
git rebase upstream/main

# push changes 
git push upstream pr_something

# or
git push origin pr_something
```

This means you can do whatever you like on main

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:137]:
```suggestion
Wiring is similar to the [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.md#connections) and other boards that follow the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
```

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (mrpollo) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:137]: although it is similar to holybros, the key selling point for NXP is the addition of ports, so I suggest NXP invest in creating a wiring diagram

**Review by mrpollo — CHANGES_REQUESTED** [review]
Its looking pretty good, i just left a few comments

**Comment by hamishwillee** [comment]
I can move it to Pixhawk standard next week, add v1.15 markup, and cross link the standard - is it likely to be ready by Tuesday?

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [reply en/flight_controller/nxp_mr_vmu_rt1176.md:137]: @dk7xe This makes sense to me, though how important this is does depend on how likely this reference design is likely to match the derivatives.

@mrpollo makes a very good point though - I wasn't even aware that the new ports were "significant" from the current documentation. To me this was mostly about the reference design benefits. So if the ports are a key difference, that would be good to capture? 
I suggest that if you agree you do this as a github suggestion.

File: en/flight_controller/index.md
Comment (hamishwillee) [inline en/flight_controller/index.md:6]: FYI, too many lists of FCs. I removed the one below and now link to it

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:3]: FYI this badge shows first version it is present in.

File: en/SUMMARY.md
Comment (hamishwillee) [inline en/SUMMARY.md:135]: Unrelated change. Not completely certain about this, but it feels odd to have all the stuff about how you mount and use the flight controller before the list of what is available.

I could have moved the FC selection topic down, but I think this is is better. Matches the index structure now too.

**Comment by hamishwillee** [comment]
OK, I have made this "pixhawk standard" and added a v1.15 version badge. Also did some structural changes that you don't particularly need to worry about.

This is waiting on response to https://github.com/PX4/PX4-user_guide/pull/3398#discussion_r1786894739 , and presumably linking information for the new board.

File: en/flight_controller/autopilot_manufacturer_supported.md
Comment (dk7xe) [reply en/flight_controller/autopilot_manufacturer_supported.md]: Ok with me

**Comment by hamishwillee** [comment]
@dk7xe Is the info for the new board published yet? @mrpollo had a comment for you in https://github.com/PX4/PX4-user_guide/pull/3398#discussion_r1786577229 too. 

Otherwise this is good to go.

**Comment by dk7xe** [comment]
@hamishwillee the DS20 standard on 6 X-RT is not out yet. We are working on the remaining parts.
@mrpollo i messaged you on discord regarding DS020

**Comment by hamishwillee** [comment]
> @hamishwillee the DS20 standard on 6 X-RT is not out yet. We are working on the remaining parts. @mrpollo i messaged you on discord regarding DS020

Thanks. I'll wait on you all then. I do think that the "special value" of this board needs to be highlighted - the extra ports are not even mentioned.

**Comment by hamishwillee** [comment]
Any progress on this?

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (igalloway) [reply en/flight_controller/nxp_mr_vmu_rt1176.md:11]: Our understanding is this was a PX4 supported reference design. We did al the initial porting etc. It is the standard board design for V6X-RT standard. WE can talk 1:1 if this is not the case.

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (igalloway) [reply en/flight_controller/nxp_mr_vmu_rt1176.md:139]: We won't be able to make a wiring diagram like Holybro does. 
We can create a simpler block diagram

**Comment by igalloway** [comment]
> > @hamishwillee the DS20 standard on 6 X-RT is not out yet. We are working on the remaining parts. @mrpollo i messaged you on discord regarding DS020
> 
> Thanks. I'll wait on you all then. I do think that the "special value" of this board needs to be highlighted - the extra ports are not even mentioned.

FWIW - the special value of this board is not "just"  CAN3 and the T1 Ethernet (and extra Secure element.
The value really is that it is a reference design from the semiconductor manufacturer, gone through all FCC/CE ROHS REACH UKCA, EMI/RFI ESD certifications and available globally with documentation to back it up.
I don't know the best way to message that? 
We want and expect others to build volume based off this reference design.

**Comment by igalloway** [comment]
@mrpollo  - Jari was contacting you directly about your V6X-RT reference questions.
In general i am hoping we can publish something ASAP, since this documentation is a precursor to it being available on NXP.com and at distribution (the launch needs the basic docs in place... in some form. i mean showing as a board in the docs, not necessarily the Standard being published.)

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:29]: :::tip
This autopilot is [supported](../flight_controller/autopilot_pixhawk_standard.md) by the PX4 maintenance and test teams.
:::

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [reply en/flight_controller/nxp_mr_vmu_rt1176.md:11]: @igalloway No, Ramon was asking why you labeled as manufacturer supported, since it is supported. I thought I'd cleaned this up. Will do so now.

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:13]:
```suggestion

```

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:24]:
```suggestion

The _MR-VMU-RT1176_ reference design is based on the Pixhawk<sup>&reg;</sup> FMUv6X-RT open standard, the latest update to the successful family of Pixhawk<sup>&reg;</sup> flight controllers.

This is NXP's open source _reference design_ for using FMUv6X-RT, and was designed and made in collaboration with industry partners<sup>&reg;</sup>, NXP's mobile robotics team and the PX4 team.
As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
It has already gone through all FCC/CE ROHS REACH UKCA, EMI/RFI ESD certifications and is available globally
Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.

![MR-VMU-RT1176 Upright Image](../../assets/flight_controller/nxp_mr-vmu-rt1176/mr-vmu-rt1176_upleft.jpg)

The board includes the same FMU module found on Pixhawk 6X-RT paired with an NXP-based carrier board.
The carrier board offers 100Base-T1 (two-wire) automotive ethernet, an NFC antenna (Connected to SE051), and a third CAN bus.
It also removes the IO processor to enable 12 PWM ports, with 8 providing Dshot capability.

This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.

```

File: en/flight_controller/nxp_mr_vmu_rt1176.md
Comment (hamishwillee) [inline en/flight_controller/nxp_mr_vmu_rt1176.md:47]: Moved this down here
```suggestion

Note that since it is a reference design, this _particular_ board is not produced in high volume.
Similar variants will be available from our licensees.

```

**Comment by hamishwillee** [comment]
> The value really is that it is a reference design from the semiconductor manufacturer, gone through all FCC/CE ROHS REACH UKCA, EMI/RFI ESD certifications and available globally with documentation to back it up.

> I don't know the best way to message that?

I think you've captured that this is a reference design and that this is expected to be taken by others into mass market. 

The information about "gone through all FCC/CE ROHS REACH UKCA, EMI/RFI ESD certifications and available globally" is not mentioned. 
If that's important you might consider the suggestions https://github.com/PX4/PX4-user_guide/pull/3398/files#r1832067247 and https://github.com/PX4/PX4-user_guide/pull/3398/files#r1832068446 - this is your marketing section, so I don't want to get in the way.

**Comment by mrpollo** [comment]
[Link to DS-020 FMUv6XRT](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-020%20Pixhawk%20Autopilot%20v6X-RT%20Standard.pdf)

**Comment by mrpollo** [comment]
[Link to DS-020 Pixhawk FMUv6XRT Open Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-020%20Pixhawk%20Autopilot%20v6X-RT%20Standard.pdf)

**Comment by hamishwillee** [comment]
@dk7xe I committed my suggested changes to the intro - you might want to check those.

If you can add this link that @mrpollo provided I can merge. Otherwise I will merge next Wednesday when I am back working on PX4 stuff.

**Comment by dk7xe** [comment]
> If that's important you might consider the suggestions https://github.com/PX4/PX4-user_guide/pull/3398/files#r1832067247 and https://github.com/PX4/PX4-user_guide/pull/3398/files#r1832068446 
> @igalloway @dk7xe Once you have had a look at those comments let me know - you don't have to merge them - its your marketing.
> 
> 

@hamishwillee thank you for incorporating @igalloway 's statement! Looks ok.
Please publish.

**Comment by hamishwillee** [comment]
OK. Added the open standard links. This should be good to go! Thanks @dk7xe @igalloway @mrpollo

**Comment by dk7xe** [comment]
Thank you @hamishwillee and @mrpollo for your great support getting that done!

**Comment by igalloway** [comment]
Thanks everyone!

**Pull request merged**
